packages/dev/core/src/Meshes/Compression/dracoCompression.ts

```diff
@@ -55,20 +55,38 @@ export interface IDracoCompressionOptions extends Pick<IDracoCodecConfiguration,
  */
 export class DracoCompression extends DracoDecoder {
     /**
-     * The configuration. Defaults to the following urls:
-     * - wasmUrl: "https://cdn.babylonjs.com/draco_wasm_wrapper_gltf.js"
-     * - wasmBinaryUrl: "https://cdn.babylonjs.com/draco_decoder_gltf.wasm"
-     * - fallbackUrl: "https://cdn.babylonjs.com/draco_decoder_gltf.js"
+     * Default configuration for the DracoCompression. Defaults to the following:
+     * - numWorkers: 50% of the available logical processors, capped to 4. If no logical processors are available, defaults to 1.
+     * - wasmUrl: `"https://cdn.babylonjs.com/draco_wasm_wrapper_gltf.js"`
+     * - wasmBinaryUrl: `"https://cdn.babylonjs.com/draco_decoder_gltf.wasm"`
+     * - fallbackUrl: `"https://cdn.babylonjs.com/draco_decoder_gltf.js"`
      */
-    public static Configuration: IDracoCompressionConfiguration = {
-        decoder: DracoDecoder.Config, // TODO: Remove this reference or update the JSDoc with warning.
-    };
+    public static override DefaultConfiguration: IDracoCodecConfiguration = { ...DracoDecoder.DefaultConfiguration }; // Use copy
 
     /**
-     * Returns true if the decoder configuration is available.
+     * @deprecated See {@link DracoCompression.DefaultConfiguration}

```

Comment:
Hmm, this whole class is deprecated right? I would think we want people to use DracoDecoder class?